browser/components/TagListItem.js

```diff
@@ -0,0 +1,24 @@
+/**
+* @fileoverview Micro component for showing TagList.
+*/
+import React, { PropTypes } from 'react'
```

Comment:
`PropTypes` is not used in this file.
Let's use it. `handleClickTagListItem` is a little bit complicated, so declaration type for it will help us to understand.

Reply:
I think that's true.
I forgot to use it ...　I will modify it to use it.